surveys/migrations/0002_change_percentage_display_text.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2016-08-04 06:03
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('surveys', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='surveyfield',
+            name='field_type',
+            field=models.CharField(choices=[('free_text', 'Character field (free text)'), ('number', 'Number'), ('percentage', 'Percentage'), ('radio', 'Choose one option'), ('checkbox', 'Choose one or more options')], default='free_text', max_length=255),
```

Comment:
Maybe the choices can be set in the form or the serializer and keep `field_type` as a simple `CharField`?